example/50-seed.yaml

```diff
@@ -59,8 +59,19 @@ spec:
         enabled: true # shoot's kube-controller-managers get scaled down when the kube-apiserver is not reachable via external DNS
       probe: # deprecated
         enabled: true # shoot's kube-controller-managers get scaled down when the kube-apiserver is not reachable via external DNS
-    excessCapacityReservation:
-      enabled: true # this seed will deploy excess-capacity-reservation pods
+  # excessCapacityReservation:
+  #  enabled: true # optional: automatically assumed 'true' if `configs` are specified.
```

Comment:
> automatically assumed 'true' if `configs` are specified.

So I can set this to `false` but still have `configs` and then get the behaviour of `enabled=true`? This seems a bit weird, doesn't it? Maybe remove it from this example here then? Also, you could adapt the doc strings in the API to mark this field as deprecated and plan for a removal date (maybe in a few releases)?

Reply:
OK, fine then. I'm not sure whether we need to put this into the example YAML. Maybe rather move it to the docs and remove it from the YAML comment? 😄